lib/python/asterisk/sipp.py

```diff
@@ -714,7 +714,7 @@ def __evaluate_scenario_results(result):
                 default_args[defarg] = ('%s/sipp/%s' % (
                     self.test_dir, default_args[defarg]))
 
-        if '-mp' not in default_args:
+        if '-min_rtp_port' not in default_args:
```

Comment:
This is not backwards compatible. Personally the SIPp I use isn't new enough to have this, so it would no longer work for me. I'm on Ubuntu 22.04, so I expect others would experience the same. Is the version shipping in distributions or has to be built manually? Is it possible to make this backwards compatible to support both based on detection of the version?